### ISSUE TITLE
[meta] 5.0.0 changelog

### ISSUE THREAD

**Issue by KubaJastrz** [ReactTraining/react-router#6643]
There is no changelog entry for `4.4.0` and https://github.com/ReactTraining/react-router/releases/latest links to version `4.3.1`.

**Comment by burnhamrobertp** [comment]
Further, despite a tagged 4.4.0 version here in github, the new version has not been published to npmjs. Not sure if this is intentional or not.

**Comment by danhilltech** [comment]
see https://github.com/ReactTraining/react-router/issues/6630#issuecomment-473973090

**Comment by victoriafrench** [comment]
Um 5.0 Changelog? Giant SemVer bumb here.

**Comment by wintercounter** [comment]
Waiting for 5.0.1 with the braking change and the valid reason for the patch bump :P

**Comment by rally25rs** [comment]
Came here to open the same issue. Bumping to `5.x` implies a breaking change, but the changelog has no mention of `5.0.0` so not sure if I will have to make any code changes in order to upgrade. 😕

**Comment by timdorr** [comment]
See the release for now: https://github.com/ReactTraining/react-router/releases/tag/v5.0.0

**Comment by rally25rs** [comment]
It might be worth mentioning; I've seen this same thing before from monorepos (IIRC it was Jest) where the packages all need to be _exactly_ the same version, but they use ranges in their dep ranges.

The bigger problem that it tends to cause (which was my issue w/ Jest) was that you can't actually pull older versions. If you say `yarn add react-router-dom@4.2.0` then it will in turn depend on `react-router@^4.2.0` which will actually then install `react-router@4.3.1` or whatever is latest; not the version you actually wanted, nor the version that matches `react-router-dom`

It would be a good idea for all the packages to not have `^` ranges in their dependencies to each other, and instead lock them to the current specific version, and have a build/release script that bumps the versions across the board for all modules and the interconnected dependencies at the same time.

(recently I've got in the habit of just no longer using version ranges at all in any of my projects. they cause more problems than they solve.)

Anyway, thanks for the link! I always forget that "releases" is a page on github 😆

**Comment by timdorr** [comment]
Yes, that's specifically why we released 5.0 and why #6644 exists.

**Comment by mjackson** [comment]
Please see https://github.com/ReactTraining/react-router/releases/tag/v5.0.0 for the full 5.0.0 changelog. There is also a blog post [that goes into a bit more detail](https://reacttraining.com/blog/react-router-v5/).

**tl;dr** Just take your React Router 4 app and start using v5. No breaking changes.

Related: Let's move away from CHANGES.md and just log stuff in GitHub releases. No reason that file needs to be versioned with everything else, and it's just one more step for people who are trying to contribute.

**Comment by timdorr** [comment]
I'm strongly in favor of that as well. We have changelog files in redux and react-redux which are just links to the releases page. And ever since GitHub added release-only watching of repos, it's basically the "right" way to do it.
